cob-stud-clients/README.asciidoc

```diff
@@ -35,9 +35,12 @@ $ sudo sed -i 's/PermitRootLogin without-password/PermitRootLogin yes/g' /etc/ss
 
 ==== Gain root rights for admin user's
 ----
+$ sudo adduser --home /home-local/Admin Admin --force-badname
+$ sudo adduser Admin sudo
 $ sudo nano /etc/sudoers
 ##add to the list###
 stud-admin ALL=(ALL) NOPASSWD: ALL
```

Comment:
yes, stud-admin has still sudo rights, but it is a pool account and not a local one. This user is also the administrator for cob-kitchen-svr